esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -3488,10 +3504,17 @@ bool CWsDfuEx::doLogicalFileSearch(IEspContext &context, IUserDescriptor* udesc,
     ForEach(*it)
         addToLogicalFileList(it->query(), NULL, version, logicalFiles);
 
-    if (version >= 1.24)
-        resp.setCacheHint(cacheHint);
+    if ((oldCacheHint != cacheHint) && (totalFiles > maxFiles))
+    {
+        VStringBuffer warning("More than %d files matched. The files are returned in random order. Using a correct filter may reduce the number of files matched.",
```

Comment:
>The files are returned in random order

Although a subset, they are still sorted aren't they ?
So I think better to say:

"The returned results represent a subset of the total number of matches".